Assets/LeapMotionModules/Interaction/InteractionHand.cs

```diff
@@ -218,8 +229,7 @@ private struct HoverCheckResults {
 
     private void FixedUpdateTouch() {
       _touchActivityManager.activationRadius = interactionManager.WorldTouchActivationRadius;
-
-      _touchActivityManager.FixedUpdateHand(_hand);
+      _touchActivityManager.FixedUpdateHand((_hand != null) ? _hand.PalmPosition.ToVector3() : Vector3.zero);
```

Comment:
Or I guess we could just have FixedUpdateHand check to see if the hand is null, and if so, clear the active list and return.